.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
You should add `.idea` to your global gitignore

Reply:
Sorry about that. Removed those.